frontend/containers/profile-header/component.tsx

```diff
@@ -0,0 +1,121 @@
+import { FC } from 'react';
+
+import { Heart as HeartIcon } from 'react-feather';
+import { FormattedMessage, useIntl } from 'react-intl';
+
+import Image from 'next/image';
+
+import { noop } from 'lodash-es/noop';
+
+import WebsiteSocialContact from 'containers/website-social-contact';
+
+import Button from 'components/button';
+import LayoutContainer from 'components/layout-container';
+
+import type { ProfileHeaderProps } from './types';
+
+export const ProfileHeader: FC<ProfileHeaderProps> = ({
```

Comment:
There are some improvements to be made on iPhone 5.

<img src="https://user-images.githubusercontent.com/6073968/158140629-a6f9c140-c9cb-476f-ac1a-d79b1bcfdbd9.png" width="320" alt="Component on iPhone 5" />